Packs/CortexXDR/Playbooks/Cortex_XDR_incident_handling_v3_6_5.yml

```diff
@@ -2016,21 +2013,7 @@ inputs:
   description: This input indicates whether the playbook will hunt for related IOCs. Specify Yes/No.
   playbookInputQuery:
 - key: InternalRange
-  value:
-    complex:
-      root: lists
-      accessor: PrivateIPs
-      transformers:
-      - operator: RegexReplace
-        args:
-          action_dt: {}
-          ignore_case: {}
-          multi_line: {}
-          output_format: {}
-          period_matches_newline: {}
-          regex:
-            value:
-              simple: IANA_Private_Address
+  value: {}
```

Comment:
Can we keep the default value for internal ranges? Why you removed that?

Reply:
Fixing